client/homebrew/pages/newPage/newPage.jsx

```diff
@@ -70,12 +73,18 @@ const NewPage = createClass({
 	},
 
 	componentDidMount : function() {
-		const storage = localStorage.getItem(KEY);
-		if(!this.props.brew.text && storage){
+		const brewStorage = localStorage.getItem(BREWKEY);
+		if(!this.props.brew.text && brewStorage){
 			this.setState({
-				brew : { text: storage }
+				brew : { text: brewStorage }
 			});
-		}
+		};
+		const styleStorage = localStorage.getItem(STYLEKEY);
+		if(!this.props.brew.style && styleStorage){
+			this.setState({
+				brew : { style: styleStorage }
+			});
+		};
```

Comment:
Unfortunately this doesn't quite work. If both localStorage keys exist, then you are first setting the state to be a brew object with only `text = brewStorage` and then immediately overwriting it with a brew object that only has `style = styleStorage`. One option is to use something like below, which you can see in other places in this page. The other option would be to just rewrite this so both parts get applied in the same `setState` call. I'm going to go ahead and do that to merge, but just FYI for the future because it comes up a lot in the Homebrewery.

```
this.setState({
  this.set_.merge(brew, {text: brewStorage});
});
```